index.rst

```diff
@@ -1,16 +1,24 @@
 OMERO guide
 ===========
 
+The OMERO guide offers step-by-step instructions on how to use components of the OMERO platform.
+To learn more about OMERO, we recommend that you first visit `https://www.openmicroscopy.org/omero/ <https://www.openmicroscopy.org/omero/>`_
+
 The OMERO guide is divided into several parts:
 
 :doc:`core_features` covers how to get started, import, manage data and run server-side OMERO scripts.
 
-:doc:`add_ons` includes instructions for installing Web, Insight and Command Line Interface plugins
+:doc:`web_addons` includes instructions for installing Web plugins
```

Comment:
This renders as "View data in OMERO.web" which doesn't quite get "add-ons" across.

Reply:
Could be replaced by "OMERO.web extensions to view data"